src/ScottPlot4/ScottPlot.WinForms/ScottPlot.WinForms.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk.WindowsDesktop">
 
     <PropertyGroup>
-        <TargetFrameworks>net6.0-windows;net5.0-windows;net472</TargetFrameworks>
+        <TargetFrameworks>net6.0-windows</TargetFrameworks>
```

Comment:
I don't think we should change the target platform here: Although this _builds_ when targeting .NET standard, all the build targets are intentional because math operations are optimized in a version-specific way (#1927)